recipes/tensorflow-datasets/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "tensorflow-datasets" %}
+{% set version = "4.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 93a89cd599b24e9cb04a3a3b43cc72728fc11889f41e9d69f0bd95a8318531bc
+
+build:
+  number: 0
+  skip: True  # [win]
+  entry_points:
+    - tfds = tensorflow_datasets.scripts.cli.main:launch_cli
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - absl-py
+    - attrs >=18.1.0
+    - dataclasses  # [py<37]
```

Comment:
These and the other requirements may be able to be unconditional. Some of these packages were converted to empty but available packages on newer Python version. Removing the selectors would have the benefit this recipe being `noarch: python`. It will fail because some dependencies are not available on Windows but for `noarch: python` we are happy to merge if CI passes on Linux.

Reply:
@xhochy thanks! I have made these updates